usr/bin/update-torbrowser

```diff
@@ -897,6 +897,10 @@ tb_preparation() {
          [ -n "$CURL_FORCE_SSL" ] || CURL_FORCE_SSL="--tlsv1.3 --proto =https --cert-status"
       fi
 
+      if [ -z "$TBB_VERSIONS_FILE_LINK" ] && [ "$tb_browser_name" = "mullvad-browser" ]; then
+         TBB_VERSIONS_FILE_LINK="https://cdn.mullvad.net/browser/update_responses/update_1/${TBB_RELEASE_CHANNEL}/downloads.json"
```

Comment:
Could you make this a variables too please?

The variable parts are:

* `https://cdn.mullvad.net/browser/update_responses/update_1`
* `https://aus1.torproject.org/torbrowser/update_3`

These could be split into two:

* `https://cdn.mullvad.net`
* `https://aus1.torproject.org`

and:

* `browser/update_responses/update_1`
* `torbrowser/update_3`

Then also clearnet vs onion needs to be considered.